src/shared/components/case-editor/case-edit-page/case-edit-page.component.ts

```diff
@@ -173,6 +173,14 @@ export class CaseEditPageComponent implements OnInit, AfterViewChecked {
     }
   }
 
+  resetFormControls(formGroup: FormGroup, caseFieldId: string): void {
```

Comment:
Can we add comment here on why we need to do this? Also, this is an intermediate solution as I understand correct?